tests/phpunit/tests/Admin/Dashboard.php

```diff
@@ -38,6 +38,13 @@ public static function wpTearDownAfterClass() {
 		self::delete_user( self::$user_id );
 	}
 
+	public function tearDown() {
+		wp_dequeue_script( \Google\Web_Stories\Admin\Dashboard::SCRIPT_HANDLE );
+		wp_dequeue_style( \Google\Web_Stories\Admin\Dashboard::SCRIPT_HANDLE );
```

Comment:
Is the `WP_UnitTestCase` not already resetting all enqueues?

Reply:
Glancing over it, it doesn't seem so.

Reply:
But good call, I can add this to `TestCase`